### PR TITLE
ENT-4694 Add Quarkus Logging Splunk dependency to swatch-producer-aws

### PR DESCRIPTION
**Pull request by lindseyburnett**
# Local Testing
- Set env vars `ENABLE_SPLUNK_HEC`, `SPLUNK_HEC_URL`, and `SPLUNK_HEC_TOKEN`.  Note that the default value for the URL that's checked in is the public-facing URL, and doesn't work over VPN.  BitWarden has a copy of an internal  URL you can use, as well as the tokens for both.

- Run gradle command
`./gradlew :swatch-producer-aws:quarkusDev`

- Query splunk to confirm existence of logs
`index="rh_rhsm" sourcetype=quarkus_service source="swatch-producer-aws"`

Confirm the following fields exist for log events: `host`, `namespace`, `source`, `sourcetype`

# EE Testing
- Get the latest image tag for this pr: https://quay.io/repository/cloudservices/rhsm-subscriptions?tab=tags and deploy using bonfire
`bonfire deploy --timeout 3600 -i quay.io/cloudservices/rhsm-subscriptions=${IMAGE_TAG} swatch-producer-aws`

- Query splunk to confirm existence of logs
`index="rh_rhsm" sourcetype=quarkus_service source="swatch-producer-aws"`

Confirm the following fields exist for log events: `host`, `namespace`, `source`, `sourcetype`

Confirm that the host/namespace fields are correctly set based on the ephemeral environment namespace you deployed to.

### CODE REVIEW

**Comment by lindseyburnett** [comment]
To get the namespace field to show up, need something along the lines of this: https://github.com/quarkiverse/quarkus-logging-splunk/pull/89

File: swatch-producer-aws/build.gradle [hunk 1]
```diff
@@ -7,52 +7,44 @@ plugins {
 }
 
 dependencies {
+    compileOnly libraries["lombok"]
```
Comment (lindseyburnett) [inline swatch-producer-aws/build.gradle:10]: almost all of these dependency changes were just reordering/alphabetizing.  then the addition of the splunk logging dependency

File: swatch-producer-aws/deploy/clowdapp.yaml [hunk 2]
```diff
@@ -15,12 +15,34 @@ parameters:
...
+    value: 'quarkus_service'
+  - name: SPLUNK_HEC_BATCH_SIZE
+    value: '10'
+  - name: SPLUNK_HEC_BATCH_INTERVAL
+    value: '10S'
+  - name: SPLUNK_HEC_RETRY_COUNT
+    value: '3'
+  - name: SPLUNK_HEC_INCLUDE_EX
+    value: 'true'
+  - name: SWATCH_INTERNAL_SUBSCRIPTION_ENDPOINT
```
Comment (lindseyburnett) [inline swatch-producer-aws/deploy/clowdapp.yaml:40]: there are subtasks on the "deploy to stage with clowder" to change these last few params

File: swatch-producer-aws/src/main/resources/application.properties [hunk 3]
```diff
@@ -47,15 +37,54 @@ kafka.bootstrap.servers=localhost:9092
 quarkus.reactive-messaging.kafka.serializer-generation.enabled=false
 
 # Consumer settings
-# Go back to the first records, if it's our first access
 mp.messaging.incoming.tally-in.connector=smallrye-kafka
 mp.messaging.incoming.tally-in.topic=platform.rhsm-subscriptions.tally
+# Go back to the first records, if it's our first access
```
Comment (lindseyburnett) [inline swatch-producer-aws/src/main/resources/application.properties:42]: moving this back down because it's only relevant to the following prop

**Review by kflahert — APPROVED** [review]

**Pull request merged**